README_ZH_CN.md

```diff
@@ -0,0 +1,111 @@
+# SigLens
+
+SigLens 是一个开源可观测性解决方案，比 Splunk 和 Elastic 高效 **100 倍**。
+
+## 为什么选择 SigLens：
+为 10,000 多名工程师提供可观测性工具的经验教会了我们一些事情：
+
+- 开发人员不得不在各种的工具之间切换以查看 logs、metrics 和 traces
+- Splunk、DataDog、NewRelic 的费用非常昂贵
+- ElasticSearch 需要太多的机器，集群维护比较困难
+- Grafana Loki 的查询性能较慢
+
+凭借几十年的监控领域经验，我们着手开始构建一个全新的，专为 logs、metrics 和 traces 而设计的，并且具有**零外部依赖**的可观测性数据库。只需一个**单一的二进制文件**，就可以在您的笔记本电脑上运行，处理每天高达 8 TB 的数据。
+
+## 加入我们的 Slack 社区
+在 [Slack](https://www.siglens.com/slack) 上和我们打个招呼吧 👋
+
+## 入门
+
+### 通过 Git 安装
+```
+git clone git@github.com:siglens/siglens
+cd siglens
+go run cmd/siglens/main.go --config server.yaml
+```
+
+### 通过 SigLens 二进制文件安装
+
+待定
+
+### 通过 Docker 安装
+
+- SigLens 可以安装在 Linux 或 macOS 机器上
+- 在 macOS 上安装时，请在运行脚本之前安装 Docker 引擎
+- 克隆 SigLens 仓库并进入 siglens 目录
+
+```
+git@github.com:siglens/siglens.git
+cd siglens
+```
+
+- 运行 `install_with_docker.sh` 脚本：
+
+```
+./install_with_docker.sh
+```
+
+SigLens 后端独立部署于 UI。 为了允许 UI 连接到后端，可以使用 docker network
+
+```
+wget "https://github.com/siglens/siglens/releases/download/${SIGLENS_VERSION}/server.yaml"
+docker pull siglens/siglens:${SIGLENS_VERSION} 
+mkdir data
+docker run -it --mount type=bind,source="$(pwd)"/data,target=/siglens/data \
+    --mount type=bind,source="$(pwd)"/server.yaml,target=/siglens/server.yaml \
+    -p 8081:8081 -p 80:80 siglens/siglens:${SIGLENS_VERSION}
+```
```

Comment:
I think you might need to prepend four spaces to each line for indentation, as the original document has done the same.